### ISSUE TITLE
TypeError: Unexpected External value as type specifier, expected string or type

### ISSUE THREAD

**Issue by mykola-mokhnach** [Koromix/koffi#142]
This error might be Node.JS bug or could be some compatibility issue between koffi and newer Node.js versions.

Steps to reproduce:
1. Install Node.JS v21.7.1 or newer
2. Install appium: `npm i -g appium`
3. Install appium-windows-driver: `appium driver install --source=npm appium-windows-driver`
4. Run appium: `appium`

Actual result:
windows driver fails to load with the below error:

```
[Appium] TypeError: Unexpected External value as type specifier, expected string or type
[Appium]     at Object.<anonymous> (D:\Trabalho Freelancer\FlowAutomation\DesktopAutomationCSharp\node_modules\appium-windows-driver\lib\commands\winapi\user32.js:102:49)
[Appium]     at Module._compile (node:internal/modules/cjs/loader:1378:14)
[Appium]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
[Appium]     at Module.load (node:internal/modules/cjs/loader:1212:32)
[Appium]     at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
[Appium]     at Module.require (node:internal/modules/cjs/loader:1237:19)
[Appium]     at require (node:internal/modules/helpers:176:18)
[Appium]     at Object.<anonymous> (D:\Trabalho Freelancer\FlowAutomation\DesktopAutomationCSharp\node_modules\appium-windows-driver\lib\commands\gestures.js:2:1)
[Appium]     at Module._compile (node:internal/modules/cjs/loader:1378:14)
[Appium]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
[Appium]     at Module.load (node:internal/modules/cjs/loader:1212:32)
[Appium]     at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
[Appium]     at Module.require (node:internal/modules/cjs/loader:1237:19)
[Appium]     at require (node:internal/modules/helpers:176:18)
[Appium]     at Object.<anonymous> (D:\Trabalho Freelancer\FlowAutomation\DesktopAutomationCSharp\node_modules\appium-windows-driver\lib\commands\index.js:9:1)
[Appium]     at Module._compile (node:internal/modules/cjs/loader:1378:14)
[Appium]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
[Appium]     at Module.load (node:internal/modules/cjs/loader:1212:32)
[Appium]     at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
[Appium]     at Module.require (node:internal/modules/cjs/loader:1237:19)
[Appium]     at require (node:internal/modules/helpers:176:18)
[Appium]     at Object.<anonymous> (D:\Trabalho Freelancer\FlowAutomation\DesktopAutomationCSharp\node_modules\appium-windows-driver\lib\driver.js:6:1)
[Appium]     at Module._compile (node:internal/modules/cjs/loader:1378:14)
[Appium]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
[Appium]     at Module.load (node:internal/modules/cjs/loader:1212:32)
[Appium]     at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
[Appium]     at Module.require (node:internal/modules/cjs/loader:1237:19)
[Appium]     at require (node:internal/modules/helpers:176:18)
```

Expected error:
The driver should be loaded without issues.

Breadcrumbs:
- The issue happens in https://github.com/appium/appium-windows-driver/blob/c73215178fe61394df529b2a865321f109d52d58/lib/commands/winapi/user32.js#L102
- The issue is NOT reproducible with the latest LTS Node.js version. 
- The issue is only related to the `koffi.union` API call. It does not affect the `koffi.struct` API though.

**Comment by chrbongardt** [comment]
Hi! Is there any update on this? It also happens with node 20.12 but not with node 20.11.

**Comment by Koromix** [comment]
It is caused by what I think is a regression in Node.js itself regarding the tagging of externals.

I'll signal them; in the meantime I've **released Koffi 2.8.1** which works around this issue and works ;)

**Comment by mykola-mokhnach** [comment]
Thank you @Koromix
